src/state/notifications/selectors.ts

```diff
@@ -21,19 +22,22 @@ export const storedEmail = createSelector(local, state => state.email);
 export const storedPin = createSelector(local, state => state.pin);
 export const failedTries = createSelector(local, state => state.failedTries);
 
-export const readableError = createSelector(notificationError, failedTries, (err, failNo) => {
-  if (err.startsWith?.(messages.requestFailed5XX)) {
+export const readableError = createSelector(notificationError, failedTries, (errorMsg, failNo) => {
+  if (errorMsg.includes(GeneralHttpError.NO_MESSAGE) || errorMsg.includes(GeneralHttpError.NO_RESPONSE)) {
     return i18n.connectionIssue.couldntConnectToServer;
   }
-  if (err === messages.invalidEmail || err === messages.invalidEmail2) {
+
+  if (errorMsg.includes(EmailNotificationsError.INVALID_EMAIL)) {
     return i18n.notifications.invalidAddressError;
   }
 
-  if (err === messages.wrongPIN && failNo < CONST.emailCodeErrorMax) {
+  if (errorMsg.includes(EmailNotificationsError.WRONG_PIN) && failNo < CONST.emailCodeErrorMax) {
     return i18n.formatString(i18n.notifications.codeError, { attemptsLeft: CONST.emailCodeErrorMax - failNo });
   }
-  return err;
+
+  return errorMsg;
```

Comment:
I couldn't figure out how to do that well. The problem is this selector is for all kinds of errors, not only for the server ones. As a result, we can't return in the last `return` statement the generic server error copy like ` i18n.connectionIssue.couldntConnectToServer` because there are other errors that might occur as well,